src/like-service/Dockerfile

```diff
@@ -2,15 +2,28 @@ FROM php:8.0-cli
 
 COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
 
-COPY . /var/www
-
 WORKDIR /var/www
 
+RUN apt update
+
+RUN apt -y install git
+
 ENV COMPOSER_ALLOW_SUPERUSER=1
 
+COPY . /var/www
+
+#This is faster because reasons https://github.com/composer/composer/issues/8205#issuecomment-507256979
+RUN php /usr/bin/composer config --global repos.packagist composer https://packagist.org
+
 RUN php /usr/bin/composer install
 
 EXPOSE 8000
 
+#Fixes error while connecting since laravel requires a key
+RUN php artisan key:generate
```

Comment:
[ISSUE] This throws an error for me
`file_get_contents(/var/www/.env): Failed to open stream: No such file or directory`
Maybe we also need some .env file here?

Reply:
.env file has been removed from "gitignore".